### PR TITLE
Update for v0.4 of helm chart

### PR DESCRIPTION
**Pull request by yuvipanda**

### CODE REVIEW

**Comment by willingc** [comment]
FIxing a RTD error https://github.com/rtfd/readthedocs.org/issues/2975 in PR #106 and then I will merge this.

File: doc/source/setup-jupyterhub.rst [hunk 1]
```diff
@@ -64,15 +64,34 @@ For the following steps, use your favorite code editor. We'll use the
...
 
-1. Let's use helm to create the instances that you configured with the
-   ``config.yaml`` file. Run this command from the directory that contains the
+1. Let's add the JupyterHub `helm repository<https://github.com/kubernetes/helm/blob/master/docs/chart_repository.md>`_
+   to your helm, so you can install JupyterHub from it. This makes it easy to refer to the JupyterHub chart
+   without having to use a long URL each time.
+
+   .. code:: bash
+
+      helm repo add jupyterhub https://jupyterhub.github.io/helm-chart/
```
Comment (willingc) [inline doc/source/setup-jupyterhub.rst]: Is this the correct URL?

**Comment by yuvipanda** [comment]
It is the correct url! It redirects you to z2jh if you hit it with a
browser. It is just the gh-pages branch of the helm-chart repo, formatted
to have the index.yaml file that helm requires.

File: doc/source/setup-jupyterhub.rst (hunk 1 above)
Comment (yuvipanda) [inline doc/source/setup-jupyterhub.rst]: It is the correct url! It redirects you to z2jh if you hit it with a browser. It is just the gh-pages branch of the helm-chart repo, formatted to have the index.yaml file that helm requires.

Comment (yuvipanda) [inline doc/source/setup-jupyterhub.rst]: I should write up a 'how to make a helm-chart release' document soon.

**Comment by choldgraf** [comment]
I'm +1 to merge

File: doc/source/setup-jupyterhub.rst (hunk 1 above)
Comment (willingc) [inline doc/source/setup-jupyterhub.rst]: Got it. Thanks for the explanation.

**Pull request merged**